PhysicsTools/BPHNano/plugins/helper.h

```diff
@@ -0,0 +1,220 @@
+// original author: RK18 team
+
+#ifndef PhysicsTools_BPHNano_helpers
+#define PhysicsTools_BPHNano_helpers
+
+#include <algorithm>
+#include <limits>
+#include <memory>
+#include <vector>
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/GeometryCommonDetAlgo/interface/GlobalError.h"
+#include "DataFormats/GeometryCommonDetAlgo/interface/Measurement1D.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/PV3DBase.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/PatCandidates/interface/CompositeCandidate.h"
+#include "Math/LorentzVector.h"
+#include "RecoVertex/KinematicFitPrimitives/interface/RefCountedKinematicTree.h"
+#include "RecoVertex/VertexPrimitives/interface/ConvertToFromReco.h"
+#include "TVector3.h"
+#include "TrackingTools/GeomPropagators/interface/AnalyticalImpactPointExtrapolator.h"
+#include "TrackingTools/IPTools/interface/IPTools.h"
+#include "TrackingTools/TransientTrack/interface/TransientTrack.h"
+
+typedef std::vector<reco::TransientTrack> TransientTrackCollection;
+
+namespace bph {
```

Comment:
```suggestion
namespace BPHNano {
```

Reply:
According to CMS policy namespaces should be in lower case (see 2.7 in https://cms-sw.github.io/cms_coding_rules.html)